client/src/main/java/com/distkv/asyncclient/DistkvAsyncDictProxy.java

```diff
@@ -103,4 +104,18 @@ public DistkvAsyncDictProxy(DistkvAsyncClient client, DistkvService service) {
         .build();
     return call(request);
   }
+
+  public CompletableFuture<DistkvProtocol.DistkvResponse> expire(String key, long expireTime) {
+    ExpireRequest expireRequest = ExpireRequest
+        .newBuilder()
+        .setExpireTime(expireTime)
+        .build();
+    DistkvProtocol.DistkvRequest request = DistkvProtocol.DistkvRequest.newBuilder()
+        .setKey(key)
+        .setRequestType(RequestType.EXPIRED_DICT)
+        .setRequest(Any.pack(expireRequest))
+        .build();
+    return call(request);
+  }
```

Comment:
Code Duplicate Here. the only difference is the request type. such as `setRequestType(RequestType.EXPIRED_STR)`
1. DistkvAsyncDictProxy.java 
1. DistkvAsyncIntProxy.java 
1. DistkvAsyncListProxy.java 
1. DistkvAsyncSetProxy.java 
1. DistkvAsyncSortedListProxy.java 
1. DistkvAsyncStringProxy.java

Reply:
This is consistent with the previous code. Although it seems to be a lot of duplicates, if it is separated and processed uniformly, it will complicate other code and reduce the consistency of the proxy interface. However, I think this problem will be solved after the subsequent optimization of the data structure.